pkgs/build-support/franz/default.nix

```diff
@@ -0,0 +1,89 @@
+{ stdenv
```

Comment:
I think `pkgs/build-support` is only for more generally useful builders. I think it would be better to put this at  `pkgs/applications/networking/instant-messengers/franz/generic.nix`.

Reply:
Fixed, thanks!